ChangeLog

```diff
@@ -14,6 +24,7 @@
 	* man/BermudanSwaption.Rmd: Modified to reflect DiscountCurve
 	option and example updated
 	
+>>>>>>> eddelbuettel/master
```

Comment:
Very helpful. I think I got it. Best when I use command line as the apps do things I do not expect. Sync merged master, my fork, and my clone. Set parent and pulled, all lined up now.

Also have travis set up on my forked repo so I can run before pull.

Getting there.